src/lib/components/ui/Button/Button.jsx

```diff
@@ -132,7 +132,10 @@ Button.propTypes = {
   id: PropTypes.string,
   label: PropTypes.string.isRequired,
   labelVisibility: PropTypes.oneOf(['all', 'desktop', 'none']),
-  loadingIcon: PropTypes.element,
+  loadingIcon: PropTypes.oneOfType([
+    PropTypes.bool,
```

Comment:
I'm not sure if I'm OK with this but if it's a common practice… To be constructive: different (yet more verbose) syntax when _using the component_ would not be the right way? Like:

```jsx
loadingIcon={isRequestPending ? <Icon icon="puppy" /> : null}
```

Sorry if we tackled this today, hope it's worth mention here, too.

Reply:
… but yes, writing ternary with `children` instead of current `condition && (<JSX>)` is awkward, especially when the JSX spans multiple lines.

Reply:
There might be better solution that I did not tested:

```
loadingIcon: PropTypes.oneOfType([
  PropTypes.component,
  PropTypes.oneOf[false],
]),
```

This should be way better because it allow component or `false`.

What do you think @mbohal? If example above works, I would like to implement it for all similar prop types that allow to accept component. I think it is only button, but i can check all of them

Reply:
> I think it is only button, but i can check all of them

It will be everywhere we have `children`…

Reply:
I think we should be compatible with React: https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

So the current solution seems good to me. If i read it correctly, it should handle:
* `true`
* `false`
* `null`
* `undefined`